### ISSUE TITLE
Some minor wrong reporting by cobbler 2.6.10  ("cobler check") on redhat 7.2

### ISSUE THREAD

**Issue by jringoot** [cobbler/cobbler#1611]
cobbler check says:
```
[root@geppetto ~]# cobbler check
The following are potential configuration items that you may want to fix:

1 : SELinux is enabled. Please review the following wiki page for details on ensuring cobbler works correctly in your SELinux environment:
    https://github.com/cobbler/cobbler/wiki/Selinux
2 : file /etc/xinetd.d/rsync does not exist
3 : reposync is not installed, need for cobbler reposync, install/upgrade yum-utils?
4 : debmirror package is not installed, it will be required to manage debian deployments and repositories

Restart cobblerd and then run 'cobbler sync' to apply changes.
```
This might be expectable

```
[root@geppetto ~]# getenforce
Permissive
```
But this a lot less: 
```
[root@geppetto ~]# locate rsyncd
/etc/rsyncd.conf
/etc/sysconfig/rsyncd
/etc/systemd/system/multi-user.target.wants/rsyncd.service
/oldcobbler/usr/share/augeas/lenses/dist/rsyncd.aug
/oldcobbler/usr/share/man/man5/rsyncd.conf.5.gz
/usr/lib/firewalld/services/rsyncd.xml
/usr/lib/systemd/system/rsyncd.service
/usr/lib/systemd/system/rsyncd.socket
/usr/lib/systemd/system/rsyncd@.service
/usr/share/augeas/lenses/dist/rsyncd.aug
/usr/share/man/fr/man5/rsyncd.conf.5.gz
/usr/share/man/man5/rsyncd.conf.5.gz
```
and this also
```
[root@geppetto ~]# which reposync
/usr/bin/reposync

```
For completeness, the versions (note that it still has the old not working URL=http://www.cobblerd.org/) :

```
[root@geppetto ~]# yum info cobbler.noarch
Loaded plugins: langpacks, product-id, search-disabled-repos, subscription-manager
Installed Packages
Name        : cobbler
Arch        : noarch
Version     : 2.6.10
Release     : 1.el7
Size        : 2.1 M
Repo        : installed
From repo   : epel
Summary     : Boot server configurator
URL         : http://www.cobblerd.org/
Licence     : GPLv2+
Description : Cobbler is a network install server.  Cobbler supports PXE, ISO
            : virtualized installs, and re-installing existing Linux machines.
            : The last two modes use a helper tool, 'koan', that integrates with
            : cobbler.  There is also a web interface 'cobbler-web'.  Cobbler's
            : advanced features include importing distributions from DVDs and rsync
            : mirrors, kickstart templating, integrated yum mirroring, and built-in
            : DHCP/DNS Management.  Cobbler has a XMLRPC API for integration with
            : other applications.

[root@geppetto ~]#
[root@geppetto ~]# cat /etc/*release
NAME="Red Hat Enterprise Linux Server"
VERSION="7.2 (Maipo)"
ID="rhel"
ID_LIKE="fedora"
VERSION_ID="7.2"
PRETTY_NAME="Red Hat Enterprise Linux Server 7.2 (Maipo)"
ANSI_COLOR="0;31"
CPE_NAME="cpe:/o:redhat:enterprise_linux:7.2:GA:server"
HOME_URL="https://www.redhat.com/"
BUG_REPORT_URL="https://bugzilla.redhat.com/"

REDHAT_BUGZILLA_PRODUCT="Red Hat Enterprise Linux 7"
REDHAT_BUGZILLA_PRODUCT_VERSION=7.2
REDHAT_SUPPORT_PRODUCT="Red Hat Enterprise Linux"
REDHAT_SUPPORT_PRODUCT_VERSION="7.2"
Red Hat Enterprise Linux Server release 7.2 (Maipo)
Red Hat Enterprise Linux Server release 7.2 (Maipo)
[root@geppetto ~]# 

```

I end with a 

**Thank you for maintaining this very useful software: cobbler!**

**Comment by jmaas** [comment]
need to make action_check aware of systemd

**Comment by jmaas** [comment]
Thanks for reporting, this fix will be in the next release (2.6.11)
